be/src/storage/rowset/zone_map_index.cpp

```diff
@@ -189,14 +189,21 @@ Status ZoneMapIndexWriterImpl<type>::finish(WritableFile* wfile, ColumnIndexMeta
     return writer.finish(meta->mutable_page_zone_maps());
 }
 
+ZoneMapIndexReader::ZoneMapIndexReader() {
+    MEM_TRACKER_SAFE_CONSUME(ExecEnv::GetInstance()->column_zonemap_index_mem_tracker(), sizeof(ZoneMapIndexReader));
```

Comment:
zonemap_mem_tracker() name is clear.

Reply:
Used to distinguish SegmentZoneMap and ColumnZoneMap